src/GUI/RevEng.Shared/Efcpt/EfcptConfigMapper.cs

```diff
@@ -37,58 +37,57 @@ public static ReverseEngineerCommandOptions ToOptions(this EfcptConfig config, s
                 throw new ArgumentNullException(nameof(configPath));
             }
 
-            int selectedToBegenerated = 0; // "all"
-            if (config.codegeneration.type == "dbcontext")
+            var selectedToBeGenerated = config.CodeGeneration.Type.ToUpperInvariant() switch
             {
-                selectedToBegenerated = 1;
-            }
-
-            if (config.codegeneration.type == "entities")
-            {
-                selectedToBegenerated = 2;
-            }
+                "DBCONTEXT" => 1,
+                "ENTITIES" => 2,
+                _ => 0, // "all"
+            };
 
             var databaseType = Providers.GetDatabaseTypeFromProvider(provider, isDacpac);
-
-            var dbContextDefaultName = config.names?.dbcontextname
-                ?? GetDbContextNameSuggestion(connectionString, databaseType);
+            var fileLayout = config.FileLayout ?? new FileLayout();
```

Comment:
I prefer to avoid empty sections that are no used to keep the file "clean"

Reply:
the section will be clean, it's just a local variable to avoid "config.FileLayout?." repetitive